sendrecv/gst/Makefile

```diff
@@ -1,6 +1,6 @@
-CC	:= gcc
-LIBS	:= $(shell pkg-config --libs --cflags gstreamer-webrtc-1.0 gstreamer-sdp-1.0 libsoup-2.4 json-glib-1.0)
-CFLAGS	:= -O0 -ggdb -Wall -fno-omit-frame-pointer
-
+CC     := gcc
+LIBS   := $(shell pkg-config --libs --cflags gstreamer-webrtc-1.0 gstreamer-sdp-1.0 libsoup-2.4 json-glib-1.0)
+CFLAGS := -O0 -ggdb -Wall -fno-omit-frame-pointer \
+          $(shell pkg-config --cflags gstreamer-1.0 glib-2.0 libsoup-2.4 json-glib-1.0 glib-2.0)
```

Comment:
The packages in the cflags list need to match the ones in the `LIBS` line.  i.e. `gstreamer-webrtc-1.0` and `gstreamer-sdp-1.0` are missing

Reply:
Oh ok, I added this. I guess `"$(CC)" $(CFLAGS) $^ $(LIBS) -o $@` needs the flags to match the libs. Do I need to make them match 1:1?

Reply:
They should match 1:1 yes.

Reply:
I made them match
```
LIBS   := $(shell pkg-config --libs --cflags glib-2.0 gstreamer-1.0 gstreamer-sdp-1.0 gstreamer-webrtc-1.0 json-glib-1.0 libsoup-2.4)
CFLAGS := -O0 -ggdb -Wall -fno-omit-frame-pointer \
		$(shell pkg-config --cflags glib-2.0 gstreamer-1.0 gstreamer-sdp-1.0 gstreamer-webrtc-1.0 json-glib-1.0 libsoup-2.4)
```